README.md

```diff
@@ -1,181 +1,93 @@
 # nvim-lspconfig
+A collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
 
-Collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
-The configurations are supported on a best-effort basis, and rely on contributions 
-from regular users to stay up-to-date.
+This repo handles automatically launching and initializing language servers that are installed on your system.
 
-This repo handles automatically launching, initializing, and configuring language servers
-that are installed on your system.
+# LSP overview
+
+Neovim supports the Language Server Protocol (LSP), which means it acts as a client to language servers and includes a Lua framework `vim.lsp` for building enhanced LSP tools. LSP facilitates features like:
+
+- go-to-definition
+- find-references
+- hover
+- completion
+- rename
+- format 
+- refactor
+
+Neovim provides an interface for all of these features, and the language server client is designed to be highly extensible to allow plugins to integrate language server features which are not yet present in Neovim core such as [**auto**-completion](https://github.com/neovim/nvim-lspconfig/wiki/Autocompletion) (as opposed to manual completion with omnifunc) and [snippet integration](https://github.com/neovim/nvim-lspconfig/wiki/Snippets).
```

Comment:
I would remove the `be highly extensible` that looks like a bit too much marketing :)

Reply:
Done in r3 PR